### PR TITLE
UHM-5317: (Restrict Orders Table to Lab or Drug orders based on context)

### PR DESCRIPTION
**Pull request by mogoodrich**
https://pihemr.atlassian.net/browse/UHM-5317

### CODE REVIEW

**Comment by mogoodrich** [comment]
Whoops, I though I had fixed all the tests but it looks like two are still failing...

**Comment by mogoodrich** [comment]
Okay, tests should be fixed!

File: app/js/components/orderEntry/OrderEntryPage.jsx [hunk 1]
```diff
@@ -304,19 +310,33 @@ export class OrderEntryPage extends PureComponent {
...
-                    <FormattedMessage
-                      id="app.orders.list"
-                      defaultMessage="Orders List"
-                      description="Orders List"
-                    />
+                    { this.props.orderType === 'laborders' ? (
+                      <FormattedMessage
+                        id="app.orders.lablist"
+                        defaultMessage="Lab Orders List"
+                        description="Drug Orders List"
```
Comment (brandones) [inline app/js/components/orderEntry/OrderEntryPage.jsx:317]:
```suggestion
                        description="Lab Orders List"
```

File: app/js/components/orderEntry/OrdersTable.jsx [hunk 2]
```diff
@@ -174,12 +174,26 @@ export class OrdersTable extends PureComponent {
...
+      return true;
+    });
+
     return (
       <React.Fragment>
-        {filteredOrders &&
-          filteredOrders.length !== 0 &&
-          filteredOrders.map(order => (
+        {ordersToDisplay &&
+          ordersToDisplay.length !== 0 &&
```
Comment (brandones) [inline app/js/components/orderEntry/OrdersTable.jsx:195]:
```suggestion
```

You don't need to check the length before mapping.

**Review by brandones — APPROVED** [review]
I don't feel like I can contribute any confidence that this works, but it looks like syntactically correct JS[X].

**Comment by mogoodrich** [comment]
Thanks @brandones , merging this in!

**Pull request merged**